com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -2882,6 +2882,12 @@ public bool SpawnWithObservers
                 set => ByteUtility.SetBit(ref m_BitField, 10, value);
             }
 
+            public bool HasInstantiationPayload
```

Comment:
This is the `SceneObject` serialization. Were you intending custom data to only be possible with in scene network objects?

Reply:
No, this also works with dynamically spawned objects. The reason the code currently sits inside SceneObject is purely practical, that’s where the serialization buffer is available.

It also allows me to check faster if the object’s prefab has a handler with instantiation data.

Moving the logic entirely outside of SceneObject is possible, but it would require shifting the read operation into AddSceneObject, separating serialization and deserialization across different layers, which could make the flow harder to follow or maintain.

Reply:
Ah, that's my bad. Leaving it inside `SceneObject` looks good for now!